### PR TITLE
RSWEB-6052: enterprise managed services

### PR DESCRIPTION
**Pull request by jhthompson**
https://jira.rax.io/browse/RSWEB-6052

### CODE REVIEW

File: styleguide/derek/examples/_office365.html [hunk 1]
```diff
@@ -2151,4 +2151,66 @@ <h4 class="center">The best part about all of our plans?<br/>
       </div>
     </div>
   </div>
-</div>
\ No newline at end of file
+
+  <!-- Enterprise Managed Services -->
+   <div class="container-fluid feature-container bg-crumpled-paper">
+
+      <div class="row">
```
Comment (SyntaxStacks) [inline styleguide/derek/examples/_office365.html]: changing this to container makes it look closer to the mock in jira

File: styleguide/derek/examples/_office365.html [hunk 2]
```diff
@@ -2151,4 +2151,66 @@ <h4 class="center">The best part about all of our plans?<br/>
...
+        <!-- Left Column -->
+        <div class="col-md-5 no-padding">
+            <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/office-365/sidebyside.png">
+        </div>
+
+        <!-- Right Column -->
+        <div class="col-md-7">
```
Comment (SyntaxStacks) [inline styleguide/derek/examples/_office365.html]: Needs a white background

File: styleguide/derek/examples/_office365.html (hunk 1 above)
Comment (amberRrucker) [inline styleguide/derek/examples/_office365.html]: backgrounds need to be defined in a fluid container. there can be a container within that though. Just so the background spans the full width.

Comment (SyntaxStacks) [inline styleguide/derek/examples/_office365.html]: yes

File: styleguide/_themes/derek/scss/_main.scss [hunk 3]
```diff
@@ -13,3 +13,17 @@
     margin-top: 89px;
   }
 }
+
+
+.feature-container {
+  h1, h2, h3 { font-family: $subhead; font-weight: 300; text-transform: none; }
```
Comment (amberRrucker) [inline styleguide/_themes/derek/scss/_main.scss]: this seems to be a duplicate from another PR

File: styleguide/derek/examples/_office365.html [hunk 4]
```diff
@@ -2151,4 +2151,66 @@ <h4 class="center">The best part about all of our plans?<br/>
...
+          <div class="col-md-11 half-padding">
```
Comment (amberRrucker) [inline styleguide/derek/examples/_office365.html]: why 11 and not 12?

File: styleguide/derek/examples/_office365.html [hunk 5]
```diff
@@ -2151,4 +2151,66 @@ <h4 class="center">The best part about all of our plans?<br/>
...
+              <h2>Managed Services for Office 365.</h2>
+              <p>Maximize your Office 365 investment with advisory services and Enterprise-class support for Office 365. Get more out of your Office 365 investment.</p>
+          </div>
+          <!-- Feature 1 -->
+          <div class="col-md-12 feature">
```
Comment (amberRrucker) [inline styleguide/derek/examples/_office365.html]: shouldnt need to create any new styling so the "feature" class is probably not necessary.

File: styleguide/derek/examples/_office365.html [hunk 6]
```diff
@@ -2151,4 +2151,66 @@ <h4 class="center">The best part about all of our plans?<br/>
...
+            <div class="col-md-2 col-sm-1">
+              <i class="rsweb gen-locked-checked-shield black"></i>
+            </div>
+            <div class="col-md-10 col-sm-11">
+              <h3>Work More Securely</h3>
+              <p>Work and share files confidently with built-in, customizable security and privacy controls that help keep your critical data safe.</p>
+            </div>
+          </div>
+          <!-- CTA -->
+          <div class="col-md-12 center">
```
Comment (amberRrucker) [inline styleguide/derek/examples/_office365.html]: Should be in a <div class="row"> then <div class="col-md-12"> center should only be on text elements.

File: styleguide/derek/examples/_office365.html [hunk 7]
```diff
@@ -2151,4 +2151,66 @@ <h4 class="center">The best part about all of our plans?<br/>
...
+            <a href="javascript:" class="button">View Plans and Pricing</a>
+          </div>
+        </div>
+      </div>
+      <div class="row bg-medium-teal">
```
Comment (amberRrucker) [inline styleguide/derek/examples/_office365.html]: rows have padding so the background wont span the full width here. If you want it to you should put the background onto a <div class="container-fluid"> which would go around the row.
